docs/server/installation.rst

```diff
@@ -388,55 +232,23 @@ software. To run it, just issue:
 
    (env) $ pootle start
 
-
```

Comment:
My preferences it to use 2 blank lines after blocks and before sections. IMHO that increases readability.